front/src/modules/rollingStock/consts.ts

```diff
@@ -223,6 +231,21 @@ export const RS_SCHEMA_PROPERTIES: readonly SchemaProperty[] = [
   },
 ];
 
+export const conversionFactorsSchema: { [key: string]: { [key: string]: number } } = {
```

Comment:
I would rather use Scream case, what do you think ? I find it more identifiable
```suggestion
export const CONVERSION_FACTORS_SCHEMA: { [key: string]: { [key: string]: number } } = {
```